aspnetcore/fundamentals/logging/index.md

```diff
@@ -624,6 +624,17 @@ Using a third-party framework is similar to using one of the built-in providers:
 
 You can create your own custom providers as well, to support other logging frameworks or your own logging requirements.
 
+## Azure Log Streaming
+
+Azure log streaming enables you to view log activity in real time from both the application and web server, as well as failed request tracing. Navigate to the **Diagnostics Logs** page from your application's portal page, turn the **Application Logging (Filesystem)** on. 
```

Comment:
Azure log streaming enables you to view log activity in real time from : 
* The application server.
* The web server.
* Failed request tracing.